### PR TITLE
bug 1464401 etcd restore clarifications

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1464401

### CODE REVIEW

File: admin_guide/backup_restore.adoc [hunk 1]
```diff
@@ -186,6 +186,11 @@ certificates in order to run pods.
 [[cluster-restore-single-member-etcd-clusters]]
 == Cluster Restore for Single-member etcd Clusters
 
+[NOTE]
+====
+The etcd v2 commands that you use in these steps are valid for both v2 and v3.
+====
+
```
Comment (e-minguez) [inline admin_guide/backup_restore.adoc]: IIRC starting with 3.6 only v3 data model is used[1], so I think v2 data is not needed.

[1] https://docs.openshift.com/container-platform/3.6/install_config/upgrading/migrating_etcd.html

File: admin_guide/backup_restore.adoc [hunk 2]
```diff
@@ -255,6 +260,11 @@ and displays {product-title}'s configuration:
 [[cluster-restore-multiple-member-etcd-clusters]]
 == Cluster Restore for Multiple-member etcd Clusters
 
+[NOTE]
+====
+The etcd v2 commands that you use in these steps are valid for both v2 and v3.
+====
```
Comment (e-minguez) [inline admin_guide/backup_restore.adoc]: Same.

**Review by e-minguez — COMMENTED** [review]
LGTM

File: admin_guide/backup_restore.adoc (hunk 1 above)
Comment (kalexand-rh) [reply admin_guide/backup_restore.adoc]: @e-minguez, interesting. I know I saw some discussion on one of my etcd bugs about you needing to migrate the v3 database even if you're still using the v2 model, but I don't recall discussion of the need to move v2 data if you were using v3. 

@wanghaoran1988, what do you think? Will you please review the rest of this change?

**Comment by kalexand-rh** [comment]
Moving 3.5 and 3.6 to here: https://github.com/openshift/openshift-docs/pull/9657

**Comment by kalexand-rh** [comment]
Actually, I think the notes can be omitted from all versions. I'm going to remove the other PR.

**Comment by kalexand-rh** [comment]
@wanghaoran1988, are you ok with this change?

**Comment by kalexand-rh** [comment]
This change was approved in the source bug.

@openshift/team-documentation, will you PTAL at this small change?

**Comment by ahardin-rh** [comment]
@kalexand-rh LGTM! :fireworks:

**Comment by kalexand-rh** [comment]
Thank you! Merging. :)

**Pull request merged**